tests/test_dataset.cpp

```diff
@@ -0,0 +1,56 @@
+#include "ismrmrd/dataset.h"
+#include "ismrmrd/ismrmrd.h"
+#include "ismrmrd/version.h"
+#include <boost/filesystem.hpp>
+#include <boost/random.hpp>
+#include <boost/test/unit_test.hpp>
+
+using namespace ISMRMRD;
+
+static boost::random::mt19937 rng;
+static boost::random::uniform_real_distribution<float> dist = boost::random::uniform_real_distribution<float>();
+
+float create_random_float() {
+    return dist(rng);
+}
+
+
+BOOST_AUTO_TEST_SUITE(DatasetTest)
+
+BOOST_AUTO_TEST_CASE(test_acquisition_header) {
```

Comment:
Think that test name is copied from the acquisition tests. Maybe choose a better name.